src/store/sagas/genres.js

```diff
@@ -0,0 +1,24 @@
+import {
+  all, call, put, takeLatest,
+} from 'redux-saga/effects';
+import genresApi from '../../api/genres';
+import { GET_GENRES_PENDING, GET_GENRES_SUCCESS, GET_GENRES_ERROR } from '../actionTypes';
+
+function* loadGenres(action) {
+  const { ids } = action.payload;
+  try {
+    const genreResponses = yield all(ids.map((id) => call(genresApi.getById, id)));
```

Comment:
so, if it array of genres, we can name it just simple `genres` (i was wrong about naming)